### ISSUE TITLE
Prophet Training Speed: Intel vs Apple M1/M2

### ISSUE THREAD

**Issue by ppiotrow** [facebook/prophet#2572]
I've noticed a significant speed gap in training time series data with Prophet: my code is roughly 5 times faster on Apple M1/M2 CPUs than on modern Intel processors. This contrast was consistent across four different setups running similar Python versions (3.11 and 3.12), with CPU architecture being the key variable.

My main question is whether this speed difference between Apple and Intel CPUs is a recognized issue? If it's not expected, could you please advise me on which dependencies I should investigate or possibly recompile to understand why Prophet runs slower on Intel CPUs compared to Apple ones?

**Comment by aakashnand** [comment]
@ppiotrow have you found any new information regarding this? I recently observed this and it seems apple silicon is definitely faster than Intel and lot of it has to do with underlying CPU architecture I guess.

**Comment by ppiotrow** [comment]
@aakashnand Yes, I did few more experiments and confirmed superiority of Apple Silicon vs Intel on friends machines.
I do not see any official claim from Prophet community and only minor mentions in Stan optimizer forum.
